jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+---
+layout: classic-docs
+title: Testing iOS Applications on macOS
+short-title: Testing iOS Applications on macOS
+categories: [configuring-jobs]
+description: Testing iOS applications on macOS
+order: 71
+---
+
+This document describes how to set up and customize testing for an iOS application with CircleCI in the following sections:
+
+* TOC
+{:toc}
+
+## Overview
+
+CircleCI offers support for building and testing iOS and macOS projects. Refer to the manifest of the software installed on CircleCI OSX and macOS build images in the Using a macOS Build Image document.
+
+## Getting Started
+
+Select a macOS project you would like to build on the [Add
+Projects page](https://circleci.com/add-projects). **Note:** If your project does not appear on the OS X tab, it may be listed as a Linux project. Add your project as a Linux build, then change
+it to a macOS build by enabling the Build OS X project option on the
+Project Settings > Build Environment page.
```

Comment:
Changing build environment is no longer needed in 2.0. If project is not listed as macOS, you can choose Linux project and then select `macOS` in the Operating System section